tests/test_hijri.py

```diff
@@ -1,26 +1,27 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
+
+import sys
+import unittest
 from datetime import datetime
 import six
-import unittest
 
 from parameterized import parameterized, param
 
-# umalqurra does not support Python3 yet
-# see https://github.com/tytkal/python-hijiri-ummalqura/pull/5
-# let's skip these tests under Python3 for now
+is_python_supported = six.PY3 and sys.version_info.minor > 5
```

Comment:
:lipstick: 

```suggestion
is_python_supported = sys.version_info >= (3, 6)
```

Slightly more readable. And we should try to avoid using `six` going forward, removing its usage should be one of the cleanup goals now that Python 2 is no longer supported.

Reply:
Totally agree!